bfdd/bfd.c

```diff
@@ -49,6 +49,8 @@ static void bs_admin_down_handler(struct bfd_session *bs, int nstate);
 static void bs_down_handler(struct bfd_session *bs, int nstate);
 static void bs_init_handler(struct bfd_session *bs, int nstate);
 static void bs_up_handler(struct bfd_session *bs, int nstate);
+static void bs_neighbour_admin_down_handler (struct bfd_session *bfd,
+						uint8_t diag);
```

Comment:
In addition to the style points that the CI reported, looks as if this isn't indented correctly?

Reply:
I am relying on CI to understand the coding styles here.
Would appreciate your help to understand what would be correct indentation in this case, i will modify code accordingly.

Reply:
the 'uint8_t' should align inside the open-paren - aligned with the 's' of 'struct' (and of course 8-space tabs, etc)

Reply:
One opening the file with VIM, with 8-space tab the 'uint8_t' is aligned with "u" of struct, this is the first position it can be after the open-paren, IMO alignment is good.

Not sure why it looks like "u" is aligned to "t" of struct in the pull request.

Reply:
sorry to pit the nit, but there it is... I'm looking at your fork, and it really is wrong: are you using 8-space tabs in your editor, for example? if you look at the nearby prototype for `bfd_find_disc`, it is a correct example - several tabs, and then a few spaces to align. you need to eliminate the space before the open-paren, and see what your tabs are doing?

Reply:
Thanks for the detailed explanation, i hope it should be good now
I am using 8 space tab, but was not adding a few spaces to align with "s" of struct